### DISCUSSION TITLE
Importsheet help needed ASAP please - BUGS!

### DISCUSSION THREAD

**Discussion by OSAwebsite** [code4recovery/12-step-meeting-list#628] [Q&A]
I am refreshing my database today and I find out the following serious shortcomings of the Importsheet. PLEASE HELP ASAP!

1. The 'types' checkboxes are not imported
2. There is no way to set the toggle YES/No for in-person meeting.
Which means I have to go through all of my 300 plus meetings by hand to add this information after an import.

What am I missing? Is this a BUG?

If the Yes/No toggle cannot be included in the sheet, at the very least include it in the bulk actions; there is only a 'set to re-open' option in there. There needs to be a 'Close for personal access' in there as well.

The types: I don't understand why these come across. When I enter them manually they do show up, even in an inactive meeting, as they should.

Thanks
Nicky

**Comment by OSAwebsite** [comment]
> set to

e-open

**Comment by OSAwebsite** [comment]
Set what to e-open?????
And is there an e-closed as well?

**Comment by OSAwebsite** [comment]
And what about the meeting types?

**Comment by OSAwebsite** [comment]
I am reverting back to a backup - actually, I restored my old files but now see that they came back in draft. Is there a way to publish all meetings all at once?

**Comment by brianw-area24** [comment]
Duplicate of Issue #633, see discussion there
